doc/api/v8.md

```diff
@@ -214,6 +214,32 @@ v8.setFlagsFromString('--trace_gc');
 setTimeout(() => { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 ```
 
+## `v8.stopCoverage()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+The `v8.stopCoverage()` method allows the user to stop the coverage collection
+started by [`NODE_V8_COVERAGE`][], so that V8 can release the execution count
+records and optimize code. This can be used in conjunction with
+`v8.takeCoverage()` if the user wants to collect the coverage on demand.
+
+## `v8.takeCoverage()`
```

Comment:
Maybe put `v8.takeCoverage()` part before v8.stopCoverage() part

Reply:
I think we usually order these alphabetically?

Reply:
`v8.serialize(value)` comes before `v8.deserialize(buffer)`, I feel this is more natural to me.